functions/find-type.py

```diff
@@ -0,0 +1,24 @@
+def find_type_arg(value):
+	"""function to find type of argument.
+
+    Args:
+       value : any type of parameter ie. int,str,float..,.
+ 
+    Returns:
+        type of value : The return value.
+        
+    Example:
+    	value  | return 
+    	2      | <type 'int'>
+
+    """	
+	print type(value)
+
+if __name__ == '__main__':
+	"""main function to get user input and call find_type_arg fuction to print type of value.
```

Comment:
You can remove arg in this conditional statement. Just you can have comment what you are doing in this condition